core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1860,8 +1856,10 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
       case _ => throw new RawlsExceptionWithErrorReport(ErrorReport(StatusCodes.BadRequest, s"Billing Account is missing or invalid for Billing Project: ${billingProject}"))
     }
 
+    val resourceBufferService = new ResourceBufferService(resourceBufferDAO, userInfo)
```

Comment:
Is this how we create services elsewhere? I'd prefer to see it get created in the constructor rather than down here